react/react1/week2/react1week2homework/src/components/TodosMethods.js

```diff
@@ -0,0 +1,82 @@
+import { TodosArray, NewTodosArray } from './TodoData';
+import React, { useState } from 'react';
+
+function TodosMethods() {
+  const [useTodo, setUseTodo] = useState(TodosArray);
```

Comment:
You'll get errors in this code because you've called this variable `useTodo`. The way React checks if something is a hook is simply by looking at if the name starts with `use`. 

So in the rest of your code React will think this is a hook and give you errors about that. 
Simply rename this to solve that, I'd suggest simply `todos`.

Reply:
Hi Milton,  i changed useTodo to todos but still does not get the checkbox ticked..

Reply:
i figured it out Milton, i passed the method as props  then it works . thank you !